xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -297,7 +301,33 @@ def _skip_target(
             "{}:{}".format(package_label_str, ctx.rule.attr.generator_name),
         )
 
+    # Set `alias_labels` only for raw `test_suite` rules by checking for the
+    # existence of the `generator_name` attribute. This attribute will be set
+    # in `ios_*_test_suite` rules.
+    if ctx.rule.kind == "test_suite" and not getattr(ctx.rule.attr, "generator_name"):
+        alias_labels = [
+            struct(
+                alias = "%s" % target.label,
+                labels = depset(
+                    [
+                        "%s" % info.label
+                        for attr, info in transitive_infos
+                        if attr == "tests"
+                    ]
```

Comment:
Explicitly finding the labels in `tests` since this is `test_suite` specific, the original intent was was to collect `info.xcode_target.label` from `deps_transitive_infos` but `info.xcode_target` is `None` for those since the targets in `tests` were skipped